plugins/de.ovgu.featureide.visualisation/src/de/ovgu/featureide/visualisation/Utils.java

```diff
@@ -36,46 +36,46 @@
 import org.osgi.framework.Bundle;
 
 /**
- * Utils for visualisations
```

Comment:
why did you change the spelling to "visuali**z**ations" when we're now using "visuali**s**ations"?